lib/ramble/docs/configuration_files.rst

```diff
@@ -561,3 +564,39 @@ format is as follows:
 
 Variables can be defined as lists, scalars, or can refer to a variable defined in
 another fully qualified experiment (through the ``cross_ref_var`` syntax).
+
+
+.. _repeats-config:
+
+------------------
+Repeats Section:
+------------------
+
+The repeats config is used to define the number of times each experiment will be
+repeated. Its format is as follows:
+
+.. code-block:: yaml
+
+    ramble:
+      config:
+        n_repeats: 'int'
+        repeats_success_strict: [True/False]
+      applications:
+        <application name>:
+          n_repeats: 'int'
+          workloads:
+            <workload name>:
+              n_repeats: 'int'
+              experiments:
+                <experiment name>:
+                  n_repeats: 'int'
+
+Repeats can be defined at the Ramble config level, which applies to all
+experiments, or at the application, workload, or experiment levels. When repeats
+are enabled, each experiment will be replicated n times and individually executed.
+Summary statistics will be calculated for each set of repeats when experiment results
+are analyzed.
+
+By default, a set of repeats is only successful if all individual experiments succeed.
+If ``repeats_success_strict`` is set to ``False``, Ramble will ignore failed experiments
+and calculate statistics for successful experiments only.
```

Comment:
It also marks t he repeat set as a success in this case, right?

Reply:
Yes, though I just double checked and found a mistake in the success criteria code, which is now fixed. Updated the verbiage here to be clear as well.